docs/resources/ui-building-blocks/components/built-in-components/pincode.md

```diff
@@ -0,0 +1,339 @@
+---
+slug: pincode
+title: PinCode
+tags: []
+description: Learn how to add PinCodewidget in your FlutterFlow app.
+sidebar_position: 4
```

Comment:
Remove sidebar position. Let this be ordered alphabetically.

Reply:
Removed!